Sources/Utilities/Storage/Session/PersistentSessionManager.swift

```diff
@@ -125,17 +129,24 @@ final class PersistentSessionManager: SessionManager {
             throw PersistentSessionError.sessionMismatch
         }
         
-        let configuration = LoginSessionConfiguration
-            .oneLogin(persistentSessionId: persistentID)
+        let sessionConfiguration: LoginSessionConfiguration = if AppEnvironment.appIntegrityEnabled {
```

Comment:
should we inject the configuration in so that the `PersistentSessionManager` supports any login configuration?

Reply:
The `AppAuthSession` only supports a `LoginSessionConfiguration` though?

Reply:
Yes, but we can have any instance of that type.